deps/deps.jl

```diff
@@ -0,0 +1,22 @@
+import Libdl
+
+libname = string(Sys.iswindows() ? "" : "lib", "xprs", ".", Libdl.dlext)
+
+if !haskey(ENV, "XPRESSDIR")
```

Comment:
I don't think we should enforce this.
Personally, I don't use it.

Reply:
Resolved in https://github.com/JuliaOpt/Xpress.jl/pull/43/commits/733a0770e340d71e0cee8848fe63705eeb5fe28a.